### PR TITLE
chore/eslint for ui-components package

### PR DESCRIPTION
**Pull request by RakeshUP**
eslint added to the ui-components package and lint errors fixed

### CODE REVIEW

File: packages/ui-components/src/components/backdrop/backdrop.tsx [hunk 1]
```diff
@@ -1,64 +1,64 @@
...
-    background:
-      'linear-gradient(167.96deg, rgba(31, 41, 51, 0.24) 0%, #1F2933 100%)',
-    transition: 'visibility 0.2s, opacity 0.2s linear',
-    backdropFilter: 'blur(24px)',
-    cursor: 'pointer',
-  };
-
-  return {className, style};
-})<StyledBackdropProps>``;
+import React, {ReactNode} from 'react';
```
Comment (WissenIstNacht) [inline packages/ui-components/src/components/backdrop/backdrop.tsx:1]: I'm confused. There's a couple of files like this one where the entire file was modified. Yet there are no apparent changes...?

**Comment by WissenIstNacht** [comment]
Hey, thanks for taking the time to do this. 

The changes themselves look good, but there's a couple things I'd like to discuss:
- How git picks up on the changes (see comment)
- Linting as npm script

//sorry, forgot to actually submit the review, haha. Find the aforementioned comment below.

File: packages/ui-components/src/components/backdrop/backdrop.tsx (hunk 1 above)
Comment (RakeshUP) [reply packages/ui-components/src/components/backdrop/backdrop.tsx:1]: It was due to the line-break character. https://eslint.org/docs/rules/linebreak-style

**Review by WissenIstNacht — APPROVED** [review]

**Review by sepehr2github — APPROVED** [review]
Thank you Rakash!

**Pull request merged**